### PR TITLE
Fix google test

### PR DESCRIPTION
**Pull request by bellenot**

### CODE REVIEW

**Comment by bellenot** [comment]
@phsft-bot build with flags -Dtesting=ON

File: cmake/modules/SearchInstalledSoftware.cmake [hunk 1]
```diff
@@ -1611,6 +1611,8 @@ if (testing)
                   -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
                   -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
                   -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
+                  -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_DEBUG:PATH=""
+                  -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_RELEASE:PATH=""
```
Comment (amadio) [inline cmake/modules/SearchInstalledSoftware.cmake]: Won't this overwrite things in a multiconfiguration build?

Comment (bellenot) [reply cmake/modules/SearchInstalledSoftware.cmake]: The goal is to have a single location for the libraries (i.e. preventing them to go in the 'Debug' or 'Release' sub-directories on Windows)

**Comment by bellenot** [comment]
@phsft-bot build with flags -Dtesting=ON

File: cmake/modules/SearchInstalledSoftware.cmake (hunk 1 above)
Comment (amadio) [reply cmake/modules/SearchInstalledSoftware.cmake]: Ok, thanks!

**Pull request merged**